packages/cli-repl/src/cli-repl.ts

```diff
@@ -62,16 +62,43 @@ class CliRepl {
       this.connect(driverUri, driverOptions);
     }
   }
+  /**
+   * Returns true if a value is a shell api type
+   *
+   * @param {any} evaluationResult - The result of evaluation
+   */
+  private isShellApiType(evaluationResult: any): boolean {
```

Comment:
question about the shell type: does it also give types to errors?

Reply:
Is a good question, for now it does not, but i have a feeling it will get more clear what does that job as soon as we figure a way to deal with evaluation of code in a unified way. We should give that a stab next.

Also even in the PR that's already duplicated code. Definitely something to look into.

Reply:
Definitely something we can start looking into later. I haven't started on error formatting yet anyways.